mulighetsrommet-tiltaksokonomi/src/main/kotlin/no/nav/tiltak/okonomi/oebs/OebsRoutes.kt

```diff
@@ -0,0 +1,93 @@
+package no.nav.tiltak.okonomi.oebs
+
+import io.ktor.http.*
+import io.ktor.server.request.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import kotlinx.serialization.Serializable
+import no.nav.mulighetsrommet.ktor.exception.StatusException
+import no.nav.mulighetsrommet.serialization.json.JsonIgnoreUnknownKeys
+import no.nav.mulighetsrommet.serializers.OebsLocalDateTimeSerializer
+import no.nav.tiltak.okonomi.service.OkonomiService
+import java.time.LocalDateTime
+
+fun Route.oebsRoutes(
+    okonomiService: OkonomiService,
+) {
+    route("/api/v1/kvittering") {
```

Comment:
Har testet ut [Type-safe routing](https://ktor.io/docs/server-resources.html) for http-endepunkter i denne appen. Greit å fortsette med det her så det er konsekvente i denne appen.